project/deps.sc

```diff
@@ -130,6 +134,7 @@ object Deps {
   def swoval          = ivy"com.swoval:file-tree-views:2.1.9"
   def testInterface   = ivy"org.scala-sbt:test-interface:1.0"
   def usingDirectives = ivy"org.virtuslab:using_directives:0.0.8"
+  def zipInputStream  = ivy"io.github.alexarchambault.scala-cli.tmp:zip-input-stream:0.1.0"
```

Comment:
I think we should add a link to a repo where this code lives with some explanation of why we are using it somewhere in out developer docs.

Reply:
Just added a comment about it here in `project/deps.sc`. (Which "developer docs" are you talking about?)